public/index.html

```diff
@@ -0,0 +1,370 @@
+                    <br>UI design, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/koreanbeef.jpg" alt="screen displaying korean beef recipe">
+                <p><strong>Korean beef</strong>
+                    <br>Webdesign, brand &#38; logotypes</p>
+            </li>
+            <li class="flex-item-2">
+                <img src="./imgs/thumbs.jpg" alt="screen displaying thumbs up and down icons">
+                <p><strong>Facebook thumbs</strong>
+                    <br>Icons design, brand &#38; logotypes</p>
+            </li>
+        </ul>
+    </section>
+    <section class="section-black">
+        <object class="clip" type="image/svg+xml" data="./imgs/clipwhite.svg"></object>
+        <a id="about"></a>
+        <h1>ABOUT</h1>
+        <div class="column-two">
+            <p class="column-left">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+            <p class="column-right">Lorem Ipsum is simply dummy text of the printing and type setting industry. Lorem Ipsum has been the industry standard dummy text since the 1500's, when an unknown printer took a galley of type and scrambled it to make a type specimen book</p>
+        </div>
+        <h2 class="subheader subheader-skills">SKILLS</h2>
+        <p class="skill-list">PHOTOSHOP</p>
+        <progress value="78" max="100"></progress>
+        <p class="skill-list">HTML/CSS</p>
+        <progress value="91" max="100"></progress>
+        <p class="skill-list">ILLUSTRATOR</p>
+        <progress value="63" max="100"></progress>
+        <p class="skill-list">PHP</p>
+        <progress value="55" max="100"></progress>
+        <p class="skill-list">WORDPRESS</p>
+        <progress value="97" max="100"></progress>
+        <p class="skills-para">Do you want be one of Avengers? Join us!</p>
+        <button class="button-contact" type="submit">CONTACT US</button>
+    </section>
+    <section class="section-white">
+        <object class="clip" type="image/svg+xml" data="./imgs/clipblack.svg"></object>
+        <a id="team"></a>
+        <h1>TEAM</h1>
+        <p>Contrary to popular belief, Lorem Ipsum is not simply random. It has roots in a piece of classical Latin literature from 45BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia</p>
+        <ul class="flex-container">
+            <li class="flex-item-4">
+                <h2>JONATHAN DOES</h2>
+                <h3 class="team-title">CEO, UI&#38;UX designer</h3>
+                <img class="face-pic" src="./imgs/jondoe.png" alt="head picture of Jonathon Doe">
+                <ul class="social-media">
+                    <li><a href=""><span class="socicon">b</span></a></li>
```

Comment:
This is using font icons wrong. DO NOT EVER put a character for a font icon in markup. I don't care what the font's Readme tells you to do, DO NOT EVER do this.

Use pseudo-content:

```
.icon-foo:before { /* or :after, as appropriate */
  font-family: "Icons";
  content: "b";
  ...
}